docs/index.html

```diff
@@ -31,6 +31,10 @@
     </p>
     <h1 class="heading-xlarge u-space-b15">MOT history API</h1>
 
+    <div class="u-margin-top-50">
+        <p class="heading-large u-space-b10">A new version of the MOT History API will be available within the next X weeks.  Sign up for it <a href="https://submit.forms.service.gov.uk/form/2382/register-for-mot-history-api/7519" target="_blank">here</a>.</p>
```

Comment:
Let's get Ben to check this content and confirm if this link should go to the register page of the documentation or straight to the registration form

Reply:
Content per screenshot in ticket

Reply:
Updated